File: C/URI/1064.c
#include <stdio.h>
 
int main() {
 
  int n, contador_positivo;
  double numero, positivo, media;
        contador_positivo = 0;
        positivo = 0;
        n = 0;
        while(n < 6)
        {
          scanf("%lf", &numero);
          if(numero > 0)
          {
                contador_positivo = contador_positivo + 1;
                positivo = positivo + numero;
          }
        n = n + 1;
        }
        media = positivo/contador_positivo;
        printf("%d valores positivos\n%.1lf\n", contador_positivo, media);

 
    return 0;
}

File: php/URI/1064.php
<?php

$positivo = 0;
$soma=0;
for($i =0; $i < 6; $i++){
    $numero = trim(fgets(STDIN));

    if($numero > 0)
    {
        $positivo++;
        $soma+=$numero;
    }
  }


$soma = $soma/$positivo;
echo $positivo." valores positivos\n";
echo "".number_format($soma, 1, ".", "", )."\n";


?>